README.rst

```diff
@@ -8,31 +8,29 @@ NuoDB Client Package
 .. contents::
 
 The NuoDB Client Package bundles the latest publicly-available NuoDB_ database client verions into a single downloadable
-file. The official versions of the NuoDB Client Package file are available from the NuoDB `GitHub Releases`_ page and from the
-NuoDB `Community Edition`_ webpage.
+file. The official versions of the NuoDB Client Package file are available from the NuoDB `GitHub Releases`_ page.
 
-You can use this project to create a package containing customized sets of NuoDB
-Client packages.
+You can also use this project to create a package containing customized sets of NuoDB Client packages.
 
 Requirements
 ------------
 
-Each client in the NuoDB Client Package will have different requirements when connecting to a NuoDB database. System requirements
-for running the NuoDB database can be found on the NuoDB documentation `System Requirements`_ page.
-
-+------------------+-----------------------------------+
-|Utility           | Requirements                      |
-+==================+===================================+
-|nuosql            |GNU/Linux or Windows               |
-+------------------+-----------------------------------+
-|nuoloader         |GNU/Linux or Windows               |
-+------------------+-----------------------------------+
-|nuodb-migrator    |Java 8 or 11                       |
-+------------------+-----------------------------------+
-|nuocmd            |Python 2 with *requests* installed |
-+------------------+-----------------------------------+
-|nuodbmgr          |Java 8 or 11                       |
-+------------------+-----------------------------------+
+Each client in the NuoDB Client Package will have different requirements when connecting to a NuoDB database. The System
+requirements for running the NuoDB database are found in the NuoDB documentation `System Requirements`_ page.
+
++------------------+----------------------------------------+
+|Utility           | Requirements                           |
++==================+========================================+
+|nuosql            |GNU/Linux or Windows                    |
++------------------+----------------------------------------+
+|nuoloader         |GNU/Linux or Windows                    |
++------------------+----------------------------------------+
+|nuodb-migrator    |Java 8 or 11                            |
++------------------+----------------------------------------+
+|nuocmd            |Python 2 or 3 with *requests* installed |
```

Comment:
The client package actually includes all dependencies for nuocmd. At least the last version of the client package that was released does. The `site-packages` directory has it and the other optional dependencies, cryptography and argcomplete.

Reply:
Also, should we be more specific by saying 2.7 or 3.8 or whatever version of Python 3 we officially support?